Expat-Journal/src/App.css

```diff
@@ -35,4 +35,4 @@
   to {
     transform: rotate(360deg);
   }
-}
+} */
```

Comment:
Is there any reason we are commenting these out rather then deleting them?
Are we planing on reusing these?